client/src/components/Details.js

```diff
@@ -25,39 +22,44 @@ export default function Details() {
       });
   }, [ID]);
 
-
-  return (
-    <div className="detailDiv">
-      <div className="p-1">
-        <div className="p-1-1">
-          <p className="p-a">{property.City}</p>
-          <p className="p-a">{property.Street}</p>
-        </div>
-        <div className="p-1-2">
-          <p className="p-a">{property.Price}</p>
-          <p className="p-a">ID: {property.ID}</p>
+    return ( property ? 
+      <div>
+        <Carousel images={property.img}/>
+        <div className="detailDiv">
+          <div className="p-1">
+            <div className="p-1-1">
+              <p className="p-a">{property.City}</p>
+              <p className="p-a">{property.Street}</p>
+            </div>
+            <div className="p-1-2">
+              <p className="p-a">{property.Price}</p>
+              <p className="p-a">ID: {property.ID}</p>
+            </div>
+          </div>
+          <hr className="hr"></hr>
+          <div className="p-1">
+            <div className="p-2-1">
+              <p className="p-m1">Year Build</p>
+              <p className="p-m1">Interior</p>
+              <p className="p-m1">Bedrooms</p>
+              <p className="p-m1">Status</p>
+            </div>
+            <div className="p-2-2">
+              <p className="p-m2">{property.YearBuild}</p>
+              <p className="p-m2">{property.Interior}</p>
+              <p className="p-m2">{property.Bedrooms}</p>
+              <p className="p-m2">{property.Status}</p>
+            </div>
+          </div>
```

Comment:
I see some potential for semantic elements like `<section>` for those groups, for example.

I still don't like the class names and what they represent... I think I can remember talking about this in an earlier review. They seem to have effects on the padding property, just judging by the name. However, they don't. Take a look at [the bootstrap docs](https://getbootstrap.com/docs/5.0/utilities/spacing/) for example